apps/fz_http/lib/fz_http/shared_validators.ex

```diff
@@ -29,7 +30,7 @@ defmodule FzHttp.SharedValidators do
     validate_change(changeset, field, fn _current_field, value ->
       value
       |> split_comma_list()
-      |> Enum.find(&(not (valid_ip?(&1) or valid_fqdn?(&1))))
+      |> Enum.find(&(not (valid_ip?(&1) or valid_fqdn?(&1) or valid_hostname?(&1))))
```

Comment:
This needed to be added to support single-word hostnames such as Docker service names like `elixir`.